addons/mail/static/src/components/chat_window_manager/chat_window_manager.scss

```diff
@@ -0,0 +1,16 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_ChatWindowManager {
+    bottom: 0;
+    right: 0;
+    display: flex;
+    flex-direction: row-reverse;
+    z-index: 1000;
```

Comment:
should be computed based on other variable ideally (depending on the goal being on top of what?)